### PR TITLE
minor corrections to manual upgrade steps

### PR DESCRIPTION
**Pull request by fermulator**
went through these instructions recently, a few tweaks based on minor errors

### CODE REVIEW

File: admin_manual/maintenance/manual_upgrade.rst [hunk 1]
```diff
@@ -36,6 +38,7 @@ Always start by making a fresh backup and disabling all 3rd party apps.
    directory to see if they are there. If not, copy them from your old ``apps/``
    directory to your new one. Make sure the directory permissions of your third
    party application directories are the same as for the other ones.
+   ## THIS APPS part IS INSUFFICIENTLY DOCUMENTED (needs better instructions)
```
Comment (MorrisJobke) [inline admin_manual/maintenance/manual_upgrade.rst]: What is missing here? It looks good for me.

Comment (fermulator) [reply admin_manual/maintenance/manual_upgrade.rst]: As a new nextcloud server sysadmin, I found it vague.  The instructions give basic direction, but ideally specific steps could be provided.

Comment (fermulator) [reply admin_manual/maintenance/manual_upgrade.rst]: (for now I think its fine; just made note of it)

**Comment by fermulator** [comment]
All good then to deliver?

**Comment by MorrisJobke** [comment]
stable14 f745c8c9 071f579d

**Pull request merged**